disaster.el

```diff
@@ -84,6 +84,14 @@
 (require 'json)
 (require 'vc)
 
+(defun disaster--arch-flags ()
+  "Select the right flags depending on the right architecture."
+  (if ((lambda () "Optimize for M1 family if true"
```

Comment:
Why should we write this with a lambda and apply it just after?

Should be written simply as:

```elisp
(if (and (eq system-type 'darwin)
         (string= "arm64" (car (split-string system-configuration "-"))))
    "-mcpu=apple-m1"
  "-march=native")
```